variables.tf

```diff
@@ -316,6 +316,12 @@ variable "enable_s3_endpoint" {
   default     = false
 }
 
+variable "s3_endpoint_policy" {
+  description = "Provide a policy for the S3 endpoint"
+  type        = string
+  default     = ""
```

Comment:
will this fail if users do not provide a policy - I think this should perhaps be `null` instead.